Source/WebCore/css/parser/CSSTokenizer.cpp

```diff
@@ -661,7 +668,7 @@ static bool isNonPrintableCodePoint(UChar cc)
 // http://dev.w3.org/csswg/css-syntax/#consume-url-token
 CSSParserToken CSSTokenizer::consumeURLToken()
 {
-    m_input.advanceUntilNonWhitespace();
+    m_input.advanceUntilNewlineOrNonWhitespace();
```

Comment:
Why is it important to stop at a newline here? Do we have a test case covering that change in behavior?